<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login </string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(12, 12, 12);</string>
  </property>
  <widget class="QPushButton" name="mainloginbutton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}



</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}
</string>
   </property>
   <property name="text">
    <string>del</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:black;
border:none;
border:1px solid black;
border-radius:20px;
font: 75 28pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_8">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>380</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
font:75 36pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(225, 225, 112);
border-radius:30px;
}
QPushButton:hover{
color:rgb(255,255,127);
background-color: rgb(238, 238, 238);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}
</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
font:75 36pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(225, 225, 112);
border-radius:30px;
}
QPushButton:hover{
color:rgb(255,255,127);
background-color: rgb(238, 238, 238);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}
</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>240</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
font:75 36pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(225, 225, 112);
border-radius:30px;
}
QPushButton:hover{
color:rgb(255,255,127);
background-color: rgb(238, 238, 238);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}
</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
font:75 36pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(225, 225, 112);
border-radius:30px;
}
QPushButton:hover{
color:rgb(255,255,127);
background-color: rgb(238, 238, 238);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_12">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
color: rgb(255,255,255);
font:75 36pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(225, 225, 112);
border-radius:30px;
}
QPushButton:hover{
color:rgb(255,255,127);
background-color: rgb(238, 238, 238);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}

</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_18">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_19">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_20">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_21">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_22">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_23">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mainloginbutton_24">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: rgb(255,255,255);
	background-color: rgb(191, 191, 191);
font:75 20pt &quot;MS Shell Dlg 2&quot;;
border-radius:30px;
}
QPushButton:hover{
color:rgb(61,61,61);
background-color: rgb(231, 231, 231);
}
QPushButton:{
color:rgb(255, 255, 255);
background-color: rgb(208,0,0);
}</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
